iphone/cli/commands/_build.js

```diff
@@ -3946,7 +3946,9 @@ iOSBuilder.prototype.createXcodeProject = function createXcodeProject(next) {
 	});
 
 	// add tiverify.framework
-	xcodeProject.removeFramework('tiverify.xcframework', { customFramework: true, embed: true });
+	// technically the `lastKNownFileType` option is wrong, but the xcode module
+	// does not properly handle `.xcframework` extensions without it yet
+	xcodeProject.removeFramework('tiverify.xcframework', { lastKnownFileType: 'wrapper.framework' });
```

Comment:
So the value for `lastKnownFileType` looks incorrect to me, wasn't sure if the comment above is what you meant? Shouldn't it be `'wrapper.xcframework'`?

Here's the line in the pbxproj file that it pertains to:
```
B112F935244E69B1005901DA /* tiverify.xcframework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.xcframework; name = tiverify.xcframework; path = ../lib/tiverify.xcframework; sourceTree = "<group>"; };
```

Reply:
Yes, the comment is exactly for that. Node `xcode` doesn't know how to handle `.xcframework` and fails to determine the correct parameters that are required to remove all bits in the Xcode project. Incorrectly specifying the `lastKnownFileType` fixes that.